### PR TITLE
deploy: merge upstream Spack v0.19.0

### PR DESCRIPTION
**Pull request by heerener**

### CODE REVIEW

**Review by matz-e — CHANGES_REQUESTED** [review]
Fix the build failures, please.

File: bluebrain/repo-patches/packages/py-opencv-python/package.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 class PyOpencvPython(PythonPackage):
     """Pre-built CPU-only OpenCV packages for Python."""
 
-    homepage = "https://pypi.org/project/opencv-python"
```
Comment (matz-e) [inline bluebrain/repo-patches/packages/py-opencv-python/package.py:13]: Homepage should still be there :)

File: bluebrain/deployment/environments/externals.yaml [hunk 2]
```diff
@@ -119,7 +117,7 @@ spack:
     # - intel-mpi
     - intel-oneapi-mkl@2021.4.0
     - ispc
-    - julia
+    - julia@1.6
```
Comment (matz-e) [inline bluebrain/deployment/environments/externals.yaml]:
```suggestion
    - julia@1.6  # As requested by Polina S.
```

**Review by matz-e — CHANGES_REQUESTED** [review]

**Comment by pramodk** [comment]
@matz-e : about the a488d7cbdd1d3e78c997f5b759acc727aafa22ee - we will need hpctoolk one way or another. So soon we will need to find a solution to this.

**Pull request merged**